app/main.py

```diff
@@ -1,3 +1,65 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> int | float:
```

Comment:
other alaso can be Distance, change everywhere

Reply:
Do You mean annotation?